docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

```diff
@@ -1,43 +1,43 @@
+"Easy to use" is one of the main concepts for OpenVINO™ API 2.0. It is about simplifying migration from different frameworks to OpenVINO, but also touches on how OpenVINO is organized, how its development tools are used, and how OpenVINO-based applications are developed and deployed.
 
-To accomplish that, we have made some changes on the installation and deployment of OpenVINO in the 2022.1 release. This guide will walk you through them.
+To accomplish that, we made some changes to the installation and deployment processes of OpenVINO in the 2022.1 release. This guide will walk you through them.
 
-## Installer Package Contains OpenVINO™ Runtime Only
+## The Installer Package Contains OpenVINO™ Runtime Only
 
-Starting from OpenVINO 2022.1, Model Optimizer, Post-Training Optimization tool and Python-based Development tools such as Open Model Zoo tools are distributed via [PyPI](https://pypi.org/project/openvino-dev/) only, and are no longer included in the OpenVINO installer package. This change has several benefits as it:
+Starting from OpenVINO 2022.1, Model Optimizer, Post-Training Optimization tool, and Python-based Development tools, such as those of Open Model Zoo, are distributed via [PyPI](https://pypi.org/project/openvino-dev/) only, and are no longer included in the OpenVINO installer package. This change has several benefits, as it:
 
-* Simplifies the user experience. In previous versions, the installation and usage of OpenVINO Development Tools differ according to the distribution type (via an OpenVINO installer or PyPI). 
-* Ensures that dependencies are handled properly via the PIP package manager and support virtual environments of development tools.
+* simplifies the user experience - in previous versions, installation and usage of OpenVINO Development Tools differed from one distribution type to another (the OpenVINO installer vs. PyPI), 
+* ensures that dependencies are handled properly via the PIP package manager and support virtual environments of development tools.
 
-The structure of OpenVINO 2022.1 installer package has been organized as below:
+The structure of the OpenVINO 2022.1 installer package has been organized as follows:
 
 - The `runtime` folder includes headers, libraries and CMake interfaces.
-- The `tools` folder contains [the compile tool](../../../tools/compile_tool/README.md), [deployment manager](../../OV_Runtime_UG/deployment/deployment-manager-tool.md) and a set of `requirements.txt` files with links to the corresponding versions of the `openvino-dev` package.
+- The `tools` folder contains [the compile tool](../../../tools/compile_tool/README.md), [deployment manager](../../OV_Runtime_UG/deployment/deployment-manager-tool.md), and a set of `requirements.txt` files with links to the corresponding versions of the `openvino-dev` package.
```

Comment:
A comma should not be added before "and <the last item>" when we describe a series of items.

Reply:
When there are 3 or more items to the list, we should totally use a comma before the final "and" :) Oxford comma FTW! ;)